### ISSUE TITLE
Stuttering CPU intensive video playback

### ISSUE THREAD

**Issue by kentherzz** [bbougot/Popcorn#213]
Im running the latest version 5.0.1 and this app is seriously CPU hogging when playing a video. I monitored it and its around 35-45% all the time, the video is stuttering a bit and I have vertical lines on the picture... What kind of video player is this that is built into the app? I tried both Haswell and Sky Lake processor machines and the result is exactly the same... Windows x64 Pro.

**Comment by bbougot** [comment]
Yes I'm aware of it, issue is tracked in ffmedialement repo : https://github.com/unosquare/ffmediaelement/issues/251`
I'm waiting for a fix prior releassing Popcorn 5.0.2

**Comment by kentherzz** [comment]
Why do you use ffmediaelement? Why not just use a more lightweighted player?
@mariodivece sorry i did not capture any sample file but I was watching the latest Tomb Raider in 1080p.

**Comment by kentherzz** [comment]
@bbougot , also it is not possible to change subtitles after you pick one while watching. Have to exit the movie and click play again.

**Comment by bbougot** [comment]
Regarding subtitle change, I'm waiting for the release of ffmedialement, the following version will allow changing subtitles on the fly.
